examples/matmul.py

```diff
@@ -1,33 +1,30 @@
 from __future__ import annotations
 
+from pathlib import Path
+
 import torch
 
-import helion
+from helion._testing import import_path
 from helion._testing import run_example
-import helion.language as hl
-
-
-# static_shapes=True gives a performance boost for matmuls
-@helion.kernel(static_shapes=True)
-def matmul(x: torch.Tensor, y: torch.Tensor) -> torch.Tensor:
-    m, k = x.size()
-    k2, n = y.size()
-    assert k == k2, f"size mismatch {k} != {k2}"
-    out = torch.empty(
-        [m, n], dtype=torch.promote_types(x.dtype, y.dtype), device=x.device
-    )
-    for tile_m, tile_n in hl.tile([m, n]):
-        acc = hl.zeros([tile_m, tile_n], dtype=torch.float32)
-        for tile_k in hl.tile(k):
-            acc = torch.addmm(acc, x[tile_m, tile_k], y[tile_k, tile_n])
-        out[tile_m, tile_n] = acc
-    return out
+
+template_module = import_path(Path(__file__).parent / "template_via_closure.py")
+matmul = template_module.matmul_with_epilogue
```

Comment:
A big point of the examples is to teach people. So we shouldn't have an example file that just imports another one.  We could either leave this as a mimimal example, or we could merge the contents of both files together and remove template_via_closure.py.

Reply:
sg - I merged the contents of both files together and remove template_via_closure.py.